<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCPlotLayout</class>
 <widget class="QWidget" name="CCPlotLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CCPlot</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/spline-chart.png</normaloff>:/icons/spline-chart.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#PlotLayout
{
	
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget_canvas" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QFrame" name="functionWidgetsGrroup">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="3">
          <widget class="QPushButton" name="btn_fullView">
           <property name="toolTip">
            <string>铺满视图</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../ccplot.qrc">
             <normaloff>:/icons/plot_icos_2/zoom.png</normaloff>:/icons/plot_icos_2/zoom.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QPushButton" name="btn_ratioView">
           <property name="toolTip">
            <string>保持比例视图</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../ccplot.qrc">
             <normaloff>:/icons/plot_icos_2/golden-ratio.png</normaloff>:/icons/plot_icos_2/golden-ratio.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QPushButton" name="btn_property">
           <property name="toolTip">
            <string>属性设置</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../ccplot.qrc">
             <normaloff>:/icons/plot_icos_2/setting.png</normaloff>:/icons/plot_icos_2/setting.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="btn_print">
           <property name="toolTip">
            <string>打印输出</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../ccplot.qrc">
             <normaloff>:/icons/plot_icos_2/print.png</normaloff>:/icons/plot_icos_2/print.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btn_choose">
           <property name="toolTip">
            <string>选择数据</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../ccplot.qrc">
             <normaloff>:/icons/plot_icos_2/selection (1).png</normaloff>:/icons/plot_icos_2/selection (1).png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btn_moveCanvas">
           <property name="toolTip">
            <string>移动画布</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../ccplot.qrc">
             <normaloff>:/icons/plot_icos_2/move.png</normaloff>:/icons/plot_icos_2/move.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QFrame" name="playWidgetsGroup">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="4">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btn_pause">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../ccplot.qrc">
             <normaloff>:/icons/plot_icos_2/pause.png</normaloff>:/icons/plot_icos_2/pause.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="btn_play">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../ccplot.qrc">
             <normaloff>:/icons/plot_icos_2/play-button-arrowhead.png</normaloff>:/icons/plot_icos_2/play-button-arrowhead.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QSlider" name="playSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ccplot.qrc"/>
  <include location="../ccplot.qrc"/>
 </resources>
 <connections/>
</ui>
